exoscale/datasource_exoscale_compute_ipaddress_test.go

```diff
@@ -0,0 +1,99 @@
+package exoscale
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccDatasourceComputeIPAddress(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+%s
+
+data "exoscale_compute_ipaddress" "ip_address" {
+  zone = "ch-gva-2"
+}`, testAccIPAddressConfigCreate),
+				ExpectError: regexp.MustCompile(`You must set at least one attribute "id", "ip_address", "tags" or "description"`),
+			},
+			{
+				Config: fmt.Sprintf(`
+%s
+
+data "exoscale_compute_ipaddress" "ip_address" {
+  zone = "ch-gva-2"
+  id   = "${exoscale_ipaddress.eip.id}"
+}`, testAccIPAddressConfigCreate),
+				Check: resource.ComposeTestCheckFunc(
+					testAccDatasourceComputeIPAddressAttributes(testAttrs{
+						"description": ValidateString(testIPDescription1),
```

Comment:
This variable doesn't exist since #35 (also applies to all other references in the file):

```suggestion
						"description": ValidateString(testDescription),
```